packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/CSharpGenTests.cs

```diff
@@ -10,32 +10,32 @@ namespace Microsoft.Generator.CSharp.Tests
 {
     public class CSharpGenTests
     {
-        // Validates that the output path is parsed correctly when provided
-        [Test]
-        public void TestGetOutputPath_OutputPathProvided()
-        {
-            var outputPath = "./outputDir";
-            var parsedOutputPath = CSharpGen.ParseGeneratedSourceOutputPath(outputPath);
-            var expectedPath = Path.Combine(outputPath, "src", "Generated");
-            var areEqual = string.Equals(expectedPath, parsedOutputPath, StringComparison.OrdinalIgnoreCase);
-
-            Assert.IsTrue(areEqual);
-
-            // append 'src' to the output path and validate that it is not appended again
-            TestOutputPathAppended(outputPath, expectedPath);
-        }
-
-        // Validates that the output path is parsed correctly when an empty string is provided
-        [Test]
-        public void TestGetConfigurationInputFilePath_DefaultPath()
-        {
-            var outputPath = "";
-            var parsedOutputPath = CSharpGen.ParseGeneratedSourceOutputPath(outputPath);
-            var expectedPath = Path.Combine("src", "Generated");
-            var areEqual = string.Equals(expectedPath, parsedOutputPath, StringComparison.OrdinalIgnoreCase);
-
-            Assert.IsTrue(areEqual);
-        }
+        //// Validates that the output path is parsed correctly when provided
```

Comment:
intentional comment?

Reply:
Oh these test cases are moved to the `ConfigurationTests` because I move the method that these test cases are testing into `Configuration`.
Let me remove these.